libs/utils/utils.js

```diff
@@ -366,16 +374,21 @@ export async function loadTemplate() {
 
 export async function loadBlock(block) {
   const name = block.classList[0];
-  const { miloLibs, codeRoot } = getConfig();
+  const { miloLibs, codeRoot, p13nBlocks } = getConfig();
+
   const base = miloLibs && MILO_BLOCKS.includes(name) ? miloLibs : codeRoot;
+  let blockPath = `${base}/blocks/${name}/${name}`;
+  if (p13nBlocks?.[name]) {
+    blockPath = `${p13nBlocks[name]}/${name}`;
+  }
```

Comment:
I would probably prefer expBlocks over `p13n`. As in, experiments blocks...
```suggestion
  if (expBlocks?.[name]) blockPath = `${expBlocks[name]}/${name}`;
```

Reply:
I've had trouble deciding between experiments and personalization.  Ultimately when doing self-serve manifests that's not an experiment, so decided to go with personalization.  I could rename to `persBlocks` or `personalizationBlocks`